packages/sirius-web/frontend/sirius-web-application/src/views/library-browser/LibraryBrowser.tsx

```diff
@@ -0,0 +1,148 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { useComponent } from '@eclipse-sirius/sirius-components-core';
+import { footerExtensionPoint } from '../../footer/FooterExtensionPoints';
+import { NavigationBar } from '../../navigationBar/NavigationBar';
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
+import { makeStyles } from 'tss-react/mui';
+import Typography from '@mui/material/Typography';
+import { MaterialReactTable, MRT_ColumnDef, MRT_PaginationState, useMaterialReactTable } from 'material-react-table';
+import { useState, useEffect, useMemo, memo } from 'react';
+import { useLibraries } from './useLibraries';
+import { GQLLibrary } from './useLibraries.types';
+import { LibrariesTableProps } from './LibraryBrowser.types';
+
+const useLibrariesViewStyle = makeStyles()((theme) => ({
+  librariesView: {
+    display: 'grid',
+    gridTemplateColumns: '1fr',
+    gridTemplateRows: 'min-content 1fr min-content',
+    minHeight: '100vh',
+  },
+  main: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(5),
+    paddingTop: theme.spacing(3),
+    paddingBottom: theme.spacing(3),
+  },
+  header: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+}));
+
+export const LibraryBrowser = () => {
+  const { classes } = useLibrariesViewStyle();
+  const { Component: Footer } = useComponent(footerExtensionPoint);
+
+  const { data: libraries, totalLibrariesCount, refreshLibraries } = useLibraries(0, 10);
```

Comment:
Why is there a pagination state here and line 77? And both do not share the same value